xml/depl_maintenance.xml

```diff
@@ -508,6 +508,262 @@
    </itemizedlist>
   </sect2>
 
+  <sect2 xml:id="sec.depl.maintenance.postgre-mariadb.upgrade">
+   <title>Preparing &postgres;-Based &productname; 7 for Upgrade</title>
+   <para>
+    Upgrading &productname; 7 is only possible when it uses &mariadb; deployed
+    with the Database &barcl;. This means that before you can proceed with
+    upgrading &productname; 7, you must migrate &postgres; to &mariadb; first. The following description covers several possible scenarios.
+   </para>
+   <sect3 xml:id="sec.postgre-mariadb.upgrade.scenario1">
+    <title>Non-HA setup or HA Setup with a 2-Node Cluster and
```

Comment:
Oh, here has to be "more than 2 node cluster". Or just Database cluster with odd number of nodes.

Reply:
But in this case, the first two sentences don't make sense:

*Before you proceed, make sure that the database cluster has an odd number of nodes. If necessary, extend the cluster as described in [...]

Or am I missing something?

Reply:
I'd remove the _Before you proceed, make sure that the database cluster has an odd number of nodes. If necessary, extend the cluster as described in_ part.

That should be covered in other sections and this one is only relevant once user has more than 2 nodes in a cluster.

BTW we're talking about database cluster. If user has more clusters (like for network or OpenStack services), we do not care about them here

Reply:
Fixed.